test/shipment_cost_test.js

```diff
@@ -0,0 +1,30 @@
+const chai = require('chai')
+const chai_http = require('chai-http')
+var should = require('should')
+var app = require('../server/index')
+var server = app.listen()
+chai.use(chai_http)
+
+describe('Sample test shipment rule', function() {
+	it('Trivial shipment rule test', function(done) { 
+		chai.request(app.listen()) 
+			.post('/shipment-cost')
+			.send({}) 
+			.end(function(err, res) { 
+				should.equal(err, null)
```

Comment:
esto seguro es una funcion de prueba copiada de algun lado, pero en general un test tendria que tener solo un 'should' (probar una sola cosa) y tener un nombre que describa mejor lo que se prueba como `it('should respond with success status code')`

Reply:
Si, es un template que encontre. tiene varios should porque comprueba varias cosas del tipo de estructura que recibe, pero basicamente prueba una sola cosa que seria el resultado que recibe, solo que a ese resultado le pregunta el tipo de dato y demas cosas. De todas maneras este test en un proximo branch sera cambiado ya que tiene como premisa que tiene una regla que devuelve un cero, pero deberia poder el test insertar una regla y luego comprobar asique el resultado de la regla que inserto. Esto en el proximo branch donde se modelen mejor las reglas se cambiara

Reply:
entiendo, solo queria aclarar la semantica del nombre del test (y la regla del único assert)